sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java

```diff
@@ -701,7 +701,7 @@ static AnalyzeHealthcareEntitiesResult getRecognizeHealthcareEntitiesResult1(Str
             IterableStream.of(Collections.emptyList()));
         final HealthcareEntity healthcareEntity3 = new HealthcareEntity();
         HealthcareEntityPropertiesHelper.setText(healthcareEntity3, "progressive");
-        HealthcareEntityPropertiesHelper.setCategory(healthcareEntity3, "ConditionQualifier");
+        HealthcareEntityPropertiesHelper.setCategory(healthcareEntity3, HealthcareEntityCategory.fromString("Course"));
```

Comment:
is this a category that is not avialable in the swagger? or why is it different?

Reply:
The category is not available in the swagger. So we can have it by extending it.

Reply:
ohhh we should push either the service team to add it to the swagger, or you can go ahead and create a PR for their swagger that adds the missing types you found